rpcserver.go

```diff
@@ -1253,6 +1254,28 @@ func (r *rpcServer) PendingChannels(ctx context.Context,
 							currentHeight
 				}
 
+				for _, htlcReport := range nurseryInfo.htlcs {
+					// TODO(conner) set incoming flag
+					// appropriately after handling incoming
+					// incubation
+					htlc := &lnrpc.PendingHTLC{
+						Incoming:       false,
+						Amount:         int64(htlcReport.amount),
```

Comment:
All amounts displayed over the RPC interface should be in _`satoshis`_. This currently displayed `mSAT`.